examples/ocmd/users.demo.json

```diff
@@ -2,7 +2,7 @@
 	{
 		"id": {
 			"opaque_id": "4c510ada-c86b-4815-8820-42cdf82c3d51",
-			"idp": "https://cernbox.cern.ch",
+			"idp": "cernbox.cern.ch",
```

Comment:
The id really should be a url. It may be compared to the oidc provider, which always is a url. Including the protocol.

Reply:
By 'id' you mean the "idp" field in a UserId object? I agree that [the cs3apis documentation](https://github.com/cs3org/cs3apis/blob/9648574fa9963be02dc3bd2700c0639ee8708492/cs3/identity/user/v1beta1/resources.proto#L36) is undecisive and seems to tend towards making it a URL, but [reva's current providerauthorizer interceptor code](https://github.com/cs3org/reva/blob/33deec960655e886fb399fc32c1524cf97d9dfa1/internal/http/interceptors/providerauthorizer/providerauthorizer.go#L87) clearly requires it to be a domain.

I'll propose a non-breaking change in the providerauthorizer code so that it strips the protocol (if present) from the user's idp string before comparing it to the provider's domain string.

I'll also create a PR on the c3apis repo to make this clearer.